src/scenes/Root/Header/Header.tsx

```diff
@@ -1,23 +1,20 @@
-import { makeStyles } from 'tss-react/mui';
+import { Box, Theme } from '@mui/material';
 import { HeaderSearch } from './HeaderSearch';
 import { ProfileToolbar } from './ProfileToolbar';
 
-const useStyles = makeStyles()(({ spacing }) => ({
-  root: {
-    padding: spacing(4, 4, 1, 4),
-    display: 'flex',
-    alignItems: 'flex-start',
-    justifyContent: 'space-between',
-  },
-}));
+const headerSx = (theme: Theme) => ({
+  padding: theme.spacing(4, 4, 1, 4),
+  justifyContent: 'space-between',
+  display: { xs: 'none', sm: 'flex' },
+  flexDirection: { xs: 'column', mobile: 'row' },
+  alignItems: { xs: 'strecth', mobile: 'flex-start' },
```

Comment:
Typo on `stretch`
`alignItems: { xs: 'stretch', mobile: 'flex-start' },`

Reply:
Good catch! I realized that the `stretch` isn't needed and can change the flexDirection from xs to sm because on xs the display is none.